zppy/templates/coupled_global.py

```diff
@@ -55,7 +56,7 @@ def add_line(year, var, year1, year2, ax, format="%4.2f", lw=1, color="b"):
     i2 = (np.abs(year - year2)).argmin()
 
     tmp = np.average(var[i1 : i2 + 1])
-    ax.plot((year[i1], year[i2]), (tmp, tmp), lw=lw, color=color)
+    ax.plot((year[i1], year[i2]), (tmp, tmp), lw=lw, color=color, label="average")
```

Comment:
The other lines in the plots weren't labeled, which I thought was confusing. I have added the label here, which also resolves #562